src/Behat/Config/Converter/UseStatementAdder.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Behat\Config\Converter;
+
+use PhpParser\Node;
+use PhpParser\NodeVisitorAbstract;
+
+class UseStatementAdder extends NodeVisitorAbstract
```

Comment:
This class, inspired by similar functionality in Rector is used to traverse all nodes in the php code looking for objects that we define using their FQN. When they are found, we replace this with the short name and make a note of this usage. This will be later used to add a list of use statements at the top of the file